std/experimental/ndslice/slice.d

```diff
@@ -1093,6 +1099,7 @@ struct Slice(size_t _N, _Range)
                      && (isPointer!_Range || is(typeof(_Range.init[size_t.init]))))
                     || is(_Range == Slice!(N1, Range1), size_t N1, Range1)))
 {
+    @trusted:
```

Comment:
All methods in `Slice` should be safe AFAIK, except the methods, which manage underlaying data directly. They are marked `@system` explicitly